README.md

```diff
@@ -110,6 +112,22 @@ etcd cluster by adding the following to the Helm install command:
 --set etcd.enabled=false --set "operator.controllerSet.dbConnectionURL=jdbc:postgresql://postgres/postgresdb?user=postgresadmin&password=admin123"
 ```
 
+### Adding snapshot support
+
+LINSTOR supports the volume snapshot feature, which is currently in beta. To use it, you need to install a cluster wide
+snapshot controller. This is done either by the cluster provider, or you can use the piraeus chart.
+
+By default, the piraeus chart will install its own snapshot controller. This can lead to conflict in some cases:
+
+* the cluster already has a snapshot controller
```

Comment:
Do you know how widespread this is? I'm just wondering if our default should be to _not_ install the snapshotter, because that is the safer option and we want to make it as easy as possible to try out the operator.

Reply:
Honestly, I have no idea :sweat_smile: 

I guess it depends on how many clusters have multiple CSI drivers configured (which I think is not that common)